src/modules/imageresizer/ui/Views/InputPage.xaml

```diff
@@ -107,14 +96,11 @@
                     <RowDefinition Height="Auto" />
                 </Grid.RowDefinitions>
 
-
                 <TextBlock
-                    HorizontalAlignment="Left"
                     VerticalAlignment="Center"
                     AutomationProperties.Name="{x:Static p:Resources.Width}"
-                    FontSize="12"
```

Comment:
There's something optically off, the W should be aligned with the ComboBox. Looks like it should be moved by 1 pixel?

Before vs. after
![image](https://github.com/microsoft/PowerToys/assets/9866362/77ec7770-9af0-49e5-b6ec-978d45a36779)